src/backend/Core/Migrations/AppDbContextModelSnapshot.cs

```diff
@@ -3408,6 +3408,10 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .HasColumnType("boolean")
                         .HasColumnName("is_canceled_by_user");
 
+                    b.Property<bool>("IsFinished")
+                        .HasColumnType("boolean")
```

Comment:
Is required, bo nie masz `bool?`

Reply:
Bo w configu dałem 
            builder.Property(c => c.IsFinished).IsRequired();
ustawiłem tak jak inne były ustawione